### ISSUE TITLE
Avoid lockout when changing the current site domein

### ISSUE THREAD

**Issue by vdboor** [ecometrica/django-multisite#3]
When the current site domein is changed, a developer can lock themselves out of the site. It requires updating the `Site` and `Alias` models manually.

Can you add a validator to prevent this from happening?

**Comment by jedie** [comment]
It's related to https://github.com/ecometrica/django-multisite/issues/33 isn't it?

**Comment by vdboor** [comment]
Yes somewhat. #33 is about about a redirect fallback, where as I propose adding a validator to avoid that situation from happening as much as possible.
